flink-runtime/src/test/scala/org/apache/flink/runtime/testingUtils/TestingUtils.scala

```diff
@@ -28,7 +28,7 @@ import com.google.common.util.concurrent.MoreExecutors
 import com.typesafe.config.ConfigFactory
 import grizzled.slf4j.Logger
 import org.apache.flink.api.common.time.Time
-import org.apache.flink.configuration.{ConfigConstants, Configuration, HighAvailabilityOptions, TaskManagerOptions}
+import org.apache.flink.configuration._
```

Comment:
@aljoscha @tillrohrmann Do we have a policy in place for scala wilcard imports (in tests)?